src/winter_onboarding_2021/fleet_management_service/handlers/fleet.clj

```diff
@@ -14,11 +15,33 @@
                     :created-by user-id}]
     (if (s/valid? ::specs/fleet-form fleet-data)
       (let [fleet-id (:fleets/id (fleet-model/create fleet-data))]
-        (->  (flash-msg "Fleet created successfully!" true)
+        (->  (utils/flash-msg "Fleet created successfully!" true)
              (merge (response/redirect (format "/fleets/%s" (str fleet-id))))))
-      (->  (flash-msg "Could not create fleet, try again!" false)
+      (->  (utils/flash-msg "Could not create fleet, try again!" false)
            (merge (response/redirect "/fleets/new"))))))
 
 (defn new [_]
   {:title "Create fleet"
    :content (views/create-fleet)})
+
+(defn assoc-managers [fleet]
```

Comment:
The function name can be better.

I think a better approach would be to have a `fleet-with-managers` function in the model namespace. Also, ideally the query to fetch the fleet and it's managers should happen in a single transaction.

Reply:
Additionally, this should be a private function.